client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
Okay, so I think this is the summary:

We keep the case_id so that we can keep track of the question, test and case numbers in case we need them in the future.

We add another id (unique_id) that ignores the numbers (in case of student additions) and make it a nicely formatted string so that extracting things is easy from it. It includes the string that Kristin and I discussed above.